Framework/CurveFitting/src/FuncMinimizers/LevenbergMarquardtMDMinimizer.cpp

```diff
@@ -70,8 +70,8 @@ bool LevenbergMarquardtMDMinimizer::iterate(size_t) {
   }
 
   if (m_mu > muMax) {
-    // m_errorString = "Failed to converge, maximum mu reached";
-    // g_log.warning() << m_errorString << '\n';
+    m_errorString = "Failed to converge, maximum mu reached.";
+    g_log.information(m_errorString);
```

Comment:
Maybe the minimizers shouldn't log their error strings at all because `Fit` repeats them in `status`?